src/patients/related-persons/RelatedPersonTab.tsx

```diff
@@ -88,14 +88,17 @@ const RelatedPersonTab = (props: Props) => {
         <div className="col-md-12">
           <Panel title={t('patient.relatedPersons.label')} color="primary" collapsible>
             {relatedPersons ? (
-              <List>
-                {relatedPersons.map((r) => (
-                  <ListItem key={r.id}>{r.fullName}</ListItem>
-                ))}
-              </List>
-            ) : (
-              <h1>Loading...</h1>
-            )}
+              (relatedPersons.length > 0) ? (
+                <List>
+                  {relatedPersons.map((r) => (
+                    <ListItem key={r.id}>{r.fullName}</ListItem>
+                  ))}
+                </List>
+                ) : (
+                  <Alert color="warning" title="No Related Persons" message="Add a related person using the button above." />
```

Comment:
The title and message need to be internationalized.

See: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/src/locales/en-US/translation.json

Reply:
I've internationalized the Related Persons tab now, hopefully I've done it correctly!

Reply:
you got it! nice work